### PR TITLE
[123] Add3

### PR DESCRIPTION
**Pull request by ewigkeit1341**
Test pull request
I don't understand how I did it

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
 # GitHubStudyRepo
-Repository for study 
+Repository for study 123
```
Comment (rasnaut) [inline README.md:2]: Good description!

**Review by rasnaut — APPROVED** [review]
Good work!

**Comment by Iryna555555** [comment]
Super

**Comment by ewigkeit1341** [comment]
Good approve!

**Pull request merged**